CONTRIBUTORS.md

```diff
@@ -310,6 +314,7 @@ implementation.
 [Whitworth University]: http://www.whitworth.edu/cms/
 [Yale University]: http://physics.yale.edu/
 
+[Dan Bonachea]: https://crd.lbl.gov/divisions/amcr/computer-science-amcr/class/members/dan-bonachea/
```

Comment:
That link target is fragile (we're decommissioning that web server Any Day Now(tm)).

Please use the shortened redirector URL I provided.
Thanks again!

```suggestion
[Dan Bonachea]: https://go.lbl.gov/dan-bonachea/
```

Reply:
Ah, sorry Dan — HPE's URL defense mechanisms made the URL completely human-unreadable in my mailer, so the way I got to it was to click through on it and then grab the result from my browser.  Didn't realize I'd changed the result by doing so.

Reply:
(and I didn't think at the time that I could snag the URL from here rather than from GitHub's notification email, obvs. :) )

Reply:
No problem @bradcray !

FWIW GitHub's "Suggested change" mechanism should give you a button in the PR UI that automatically creates a commit with the reviewer's suggested change, and automatically credits the reviewer as a co-author on that commit (not that I care at all in this case, but it's nice when a reviewer provides a technical contribution).